bedshift/bedshift.py

```diff
@@ -423,9 +423,14 @@ def main():
             _LOGGER.error("You must provide either chrom sizes or a refgenie genome.")
             sys.exit(1)
         else:
-            import refgenconf
-            rgc = refgenconf.RefGenConf(refgenconf.select_genome_config())
-            args.chrom_lengths = rgc.seek(args.genome, "fasta", "", "chrom_sizes")
+            try:
+                import refgenconf
+                rgc = refgenconf.RefGenConf(refgenconf.select_genome_config())
+                args.chrom_lengths = rgc.seek(args.genome, "fasta", "", "chrom_sizes")
```

Comment:
did you test this? At first glance, it might not work. The third argument is `tag` and it's set to an empty string. You can just leave it out  or set to `None` if you want `seek` to use the default tag for this asset.

Reply:
it *does* work... but probably better to do as you suggest!

Reply:
ah right! that's because empty str are considered `False` in boolean context